lib/models/user-store.js

```diff
@@ -0,0 +1,63 @@
+// This is a guess about what a reasonable value is. Can adjust if performance is poor.
+const MAX_COMMITS = 5000;
```

Comment:
Have you looked at waterfall timelines for repositories that have more than 5k commits to see what this impact is? I doubt it'd be significant but it'd be nice to see what kind of times we're seeing at the upper cap. Especially before you get a new laptop that's less haunted :wink: